app/Http/Controllers/Auth/LoginController.php

```diff
@@ -85,53 +118,6 @@ private function loginViaRemoteUser(Request $request)
         }
     }
 
-    private function loginViaLdap(Request $request)
```

Comment:
Since the old `loginViaLdap` has the same signature of `loginByLdap` - why not just keep the old name? Just decreases the radius of change we have to figure out here.

Reply:
Revered function name to old value.